src/java.base/share/classes/sun/security/provider/DSA.java

```diff
@@ -49,9 +49,11 @@
  *
  * This file contains both the signature implementation for the
  * commonly used SHA1withDSA (DSS), SHA224withDSA, SHA256withDSA,
- * as well as RawDSA, used by TLS among others. RawDSA expects
- * the 20 byte SHA-1 digest as input via update rather than the
- * original data like other signature implementations.
+ * SHA384withDSA, SHA512withDSA, SHA3-224withDSA, SHA3-256withDSA,
+ * SHA3-384withDSA, SHA3-512withDSA, as well as RawDSA, used by
+ * TLS among others. RawDSA expects the 20 byte SHA-1 digest as
+ * input via update rather than the original data like other signature
+ * implementations.
```

Comment:
As you were already there, do you want to mention the IEEE P1363 format in the class description?  Maybe in the 1st paragraph in the class spec.

Reply:
Ok~